packages/core/saved-objects/core-saved-objects-migration-server-internal/src/initial_state.ts

```diff
@@ -94,6 +94,9 @@ export const createInitialState = ({
         dynamic: 'true',
         type: 'object',
       },
+      typeMigrationVersion: {
```

Comment:
We can remove the `migrationVersion` because we transform documents before writing them into the temp index.